penny_university_frontend/package.json

```diff
@@ -42,7 +42,7 @@
     "lint:fix": "yarn lint --fix",
```

Comment:
Is it necessary to use `yarn` over `npm`?

Reply:
oh no, that is just what I have setup and use out of habit